assets/css/custom.css

```diff
@@ -0,0 +1,8 @@
+@media (max-width: 768px) {
```

Comment:
@edykim 
The value in Bootstrap CSS is 767px.
Isn't there any problem unlike Bootstrap CSS?
https://github.com/Pluu/pluu.github.io/blob/master/assets/css/bootstrap.css#L1585

Reply:
Thanks for pointing out, I updated the number as you suggested.